modules/cluster-logging-deploy-cli.adoc

```diff
@@ -373,31 +282,68 @@ elasticsearch-cdm-x6kdekli-3   1/1     1            0           6m44s
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                            DISPLAY                  VERSION               REPLACES   PHASE
+default                                                 elasticsearch-operator.5.1.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-node-lease                                         elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-public                                             elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-system                                             elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-apiserver-operator                            elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-apiserver                                     elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-authentication-operator                       elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-authentication                                elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+...
+----
++
+There should be an {es-op} instance in each namespace. The version number might be different than shown.
 
-. Verify the installation by listing the pods in the *openshift-logging* project.
+. Verify the {clo} installation.
 +
-You should see several pods for components of the Logging subsystem, similar to the following list:
+There should be a Red Hat OpenShift Logging Operator in the `openshift-logging` namespace. The Version number might be different than shown.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
+...
+openshift-logging                                       clusterlogging.5.1.0-202007012112.p0         OpenShift Logging          5.1.0-202007012112.p0              Succeeded
+...
+----
+
+. Verify the installation by listing the pods in the *openshift-logging* project. Run the following command:
 +
 [source,terminal]
 ----
 $ oc get pods -n openshift-logging
 ----
 +
+You should see several pods for components of the {logging}, similar to the following list:
++
 .Example output
```

Comment:
Out of scope of this PR, but the output is also not aligned once you get to the collector-587vb row (and following rows).